README.md

```diff
@@ -78,7 +78,7 @@ Please make sure you’ve followed the guidelines in
     ```
 
   2. After you have cloned the EarlGrey repository, download all the dependencies using the
-  **setup-earlgrey.sh** (run the script from the cloned repo) script in the `Scripts/` directory.
+  [**setup-earlgrey.sh**](./EarlGrey/Scripts/setup-earlgrey.sh) (run the script from the cloned repo) script in the `Scripts/` directory.
```

Comment:
Thanks. @SuriyaaKudoIsc, use the `View` button to preview how the change looks on Github and make sure the link works.